inlong-tubemq/tubemq-manager/pom.xml

```diff
@@ -176,6 +176,10 @@
             <groupId>jakarta.validation</groupId>
             <artifactId>jakarta.validation-api</artifactId>
         </dependency>
+      <dependency>
```

Comment:
Need to add the LICENSE and NOTICE (if any) that the third party depends on in [1]

https://github.com/apache/inlong/blob/master/licenses/inlong-tubemq-manager/